docs/running-genesis-node.md

```diff
@@ -0,0 +1,161 @@
+## Running Node
+
+This document describes in details how to configure a genesis (first) validator node.
+
+### Hardware requirements
+
+Minimal:
+- 1GB RAM
+- 25GB of disk space
+- 1.4 GHz CPU
+
+Recommended (for highload applications):
+- 2GB RAM
+- 100GB SSD
+- x64 2.0 GHz 2v CPU
+
+### Operating System
+
+Current delivery is compiled and tested under `Ubuntu 18.04.3 LTS` so we recommend using this distribution for now. In future, it will be possible to compile the application for a wide range of operating systems thanks to Go language.
+
+## Components
+
+The delivery must consist of the following components:
+
+* Binary artifacts:
+    * dcld: The binary used for running a node.
+    * dclcli: The binary that allow users to interact with pool ledger.
+* Genesis transactions file: `genesis.json`
+* The list of alive peers: `persistent_peers.txt`. It has the following format: `<node id>@<node ip>,<node2 id>@<node2 ip>,...`.
+* The service configuration file `dcld.service`
+
+### Deployment steps
+
+1. Put `dcld` and `dclcli` binaries to `/usr/bin/` and configure permissions.
+
+2. Configure dclcli:
+    * `dclcli config chain-id dclchain`
+    * `dclcli config output json` - Output format (text/json).
+    * `dclcli config indent true` - Add indent to JSON response.
+    * `dclcli config trust-node false` - Verify proofs for node responses.
+    * `dclcli config node tcp://localhost:26657` - Address of the genesis node. 
+
+3. Prepare keys:
+    * Derive a new private key and encrypt to disk: `dclcli keys add <name>`.
+    Expected output format: 
+        ```json
+        {
+          "name": <name>, // key name. can be used for signing transactions
+          "type": "local",
+          "address": string, // bench32 encoded address
+          "pubkey": string, // bench32 encoded public key
+          "mnemonic": string // seed that can be used to generate the same private/public key pair
+        }
+        ```
+    * Remember generated `address` and `pubkey` they will be used later. 
+    You can retrieve `address` and `pubkey` values anytime using `dclcli keys show <name>`. 
+    Of course, only on the machine where the keypair was generated.
+
+4. Prepare genesis node configuration:
+
+    * Initialize new configuration: `dcld init <node-name> --chain-id dclchain`.
+    * Add genesis account with the generated key and `Trustee`, `NodeAdmin` roles:
```

Comment:
For the Test Net it can be fine, but for other Nets we may not want to mix NodeAdmin and Trustee accounts. Probably we should mention this explicitly.